components/utils/cachedrequestloader/src/main/java/org/mozilla/cachedrequestloader/BackgroundCachedRequestLoader.java

```diff
@@ -50,6 +61,9 @@ public ResponseData getStringLiveData() {
     private void loadFromCache() {
         backgroundExecutorService.submit(() -> {
             try {
+                if (delayCacheLoad) {
+                    Thread.sleep(Inject.getTestDelay(delayCacheLoad));
```

Comment:
can we  make this line like this:

`Inject.delayForTesting();`

and put `Thread.sleep(1000) `in Inject.java
```
void delayForTesting(
    Thread.sleep(1000);
}
```

so no one will accidentally run to Thread.sleep()... in production code

Reply:
Fixed.